### PR TITLE
Added config command to docs and README

### PR DESCRIPTION
**Pull request by ArjixGamer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,16 +104,18 @@ If you have trouble installing, see extended installation instructions [here](ht
 
 See [docs](https://anime-downlader.readthedocs.io/en/latest/usage/dl.html).
 
-Anime Downloader has two sub-commands, `dl` and `watch`.
+Anime Downloader has three sub-commands, `dl` and `watch`.
```
Comment (IguanasInPyjamas) [inline README.md:107]: no it doesn't

Comment (IguanasInPyjamas) [reply README.md:107]: 3 =/= 2, there's also five commands

File: README.md [hunk 2]
```diff
@@ -104,16 +104,18 @@ If you have trouble installing, see extended installation instructions [here](ht
...
 
 - [dl](https://anime-downlader.readthedocs.io/en/latest/usage/dl.html): `dl` can download anime.
 - [watch](https://anime-downlader.readthedocs.io/en/latest/usage/watch.html): `watch` can manage your anime watch list. Needs [mpv](https://mpv.io).
+- [config](https://anime-downlader.readthedocs.io/en/latest/usage/config.html): can manage the configuration for anime dl.
```
Comment (IguanasInPyjamas) [inline README.md:112]: anime test, anime ezdl

File: docs/usage/config.rst [hunk 3]
```diff
@@ -1,19 +1,84 @@
 Config
 ------
 
-You can configure the tool to suit your needs by using ``config.json``.
+You can configure the tool to suit your needs by using ``anime config``.
```
Comment (IguanasInPyjamas) [inline docs/usage/config.rst:4]: "anime config or editing the config.json"

File: docs/usage/config.rst [hunk 4]
```diff
@@ -1,19 +1,84 @@
...
 
-This can be found under:
+This command lets you change the configuration for anime dl from within your terminal.
+example:
```
Comment (IguanasInPyjamas) [inline docs/usage/config.rst:7]: Example:

File: docs/usage/config.rst [hunk 5]
```diff
@@ -1,19 +1,84 @@
...
+    |      2 | ezdl       |
+    |      1 | dl         |
+    |--------+------------|
+    |   SlNo | settings   |
+    +--------+------------+
+    Select Option [1]:
+
+
+
+It lists all options in the config, and lets you change their values by entering them.
```
Comment (IguanasInPyjamas) [inline docs/usage/config.rst:21]: This lists

File: docs/usage/config.rst [hunk 6]
```diff
@@ -1,19 +1,84 @@
...
+If you select 1 (one) then it will bring all the sub-keys for that selected choice
```
Comment (IguanasInPyjamas) [inline docs/usage/config.rst:22]: full stop

File: docs/usage/config.rst [hunk 7]
```diff
@@ -1,19 +1,84 @@
...
+    |      1 | aria2c_for_torrents                |
+    |--------+------------------------------------|
+    |   SlNo | dl settings                        |
+    +--------+------------------------------------+
+    Select Option [1]:
+    Current value: False
+    Input new value for aria2c_for_torrents: True
+
+If you want you can directly edit the config.json.
+It can be found under:
```
Comment (IguanasInPyjamas) [inline docs/usage/config.rst:74]: This is a nice addition, config.json should also be mentioned earlier.

**Review by IguanasInPyjamas — COMMENTED** [review]
Some minor mistakes. Fix them and then I'll merge.

File: docs/usage/config.rst [hunk 8]
```diff
@@ -1,10 +1,17 @@
...
+
+
+anime config
+------
+
+
 
 This command lets you change the configuration for anime dl from within your terminal.
-example:
+Example\:
```
Comment (IguanasInPyjamas) [inline docs/usage/config.rst]: "\"?

**Comment by IguanasInPyjamas** [comment]
The code blocks seem to be broken in rich view mode?

**Comment by ArjixGamer** [comment]
yeah, I'm trying to fix some things

**Comment by ArjixGamer** [comment]
I give up, rst is way too complex for the simple things.
But most of the PR is done, what remains is to fix the rst.
@IguanasInPyjamas If you can do just that it would be helpful.

**Review by IguanasInPyjamas — APPROVED** [review]

**Pull request merged**